app/javascript/containers/NavigationBar/BusMap.jsx

```diff
@@ -30,8 +33,9 @@ class BusMap extends React.Component {
     const upcoming = this.props.upcoming.map((categoryInfo) => {
       return (<InactiveCategoryNode label={categoryInfo.label}
                                     path={categoryInfo.path}
-                                    name={categoryInfo.name} />);
-    });
+                                    name={categoryInfo.name}
+                                    isDisabled={ this.props.numUnanswered > 0 && categoryInfo.name === "Review and Submit"} />)
```

Comment:
Can you put the boolean logic inside a separate helper function all call that instead?